installer/install.iss

```diff
@@ -3366,6 +3368,39 @@ begin
         InstallWindowsTerminalFragment();
     end;
 
+    {
+        Install Posh-Git from the PSGallery
+    }
+
+    if (IsComponentSelected('poshgit')) then begin
+        WizardForm.StatusLabel.Caption:='Installing Posh-Git from the PSGallery';
+        // Must use the sysnative version of PowerShell, otherwise will target the wrong profile
+        Cmd:='"'+ExpandConstant('{sysnative}\WindowsPowerShell\v1.0\powershell.exe')+'"'+
+            ' -ExecutionPolicy Bypass -NoLogo -NonInteractive -WindowStyle Hidden -command "'+
+            'if (!(Get-PackageProvider -Name NuGet)) {'+
+            ' Install-PackageProvider -Name NuGet -Force ' +
+            '} ' +
+            '$policy = (Get-PSRepository -Name PSGallery).InstallationPolicy; ' +
+            'if ($policy -ne """Trusted""") {' +
+            ' Set-PSRepository -Name PSGallery -InstallationPolicy Trusted '+
+            '} ' +
+            'Install-Module -Repository PSGallery -Scope AllUsers posh-git; ' +
+            '$res=$?; ' +
+            'if ($policy -ne """Trusted""") {' +
+            ' Set-PSRepository -Name PSGallery -InstallationPolicy $policy ' +
+            '} ' +
+            'if ($res) {' +
+            ' Add-PoshGitToProfile -AllUsers; ' +
```

Comment:
This includes a check if posh-git is already imported in _any_ of the current user's profiles, which would prevent the expected system-wide install here. It would seem reasonable for posh-git to be changed to only test the file to which the import would be added.

However, when testing this I also noticed that importing posh-git from two different locations (e.g. from the system install and then from a user install) can lead to unexpected behavior. So maybe better to err on the side of leaving all profiles alone if there's evidence of posh-git?

(No change required, here. Just thinking out loud about how posh-git should behave.)